feature/settings/src/main/java/com/google/jetpackcamera/settings/SettingsUiState.kt

```diff
@@ -29,6 +29,20 @@ import com.google.jetpackcamera.settings.ui.FPS_UNSUPPORTED_TAG
 import com.google.jetpackcamera.settings.ui.LENS_UNSUPPORTED_TAG
 import com.google.jetpackcamera.settings.ui.STABILIZATION_UNSUPPORTED_TAG
 
+const val STABILIZATION_SETTING_PREFIX = "Stabilization"
+
+const val FPS_AUTO = 0
+const val FPS_15 = 15
+const val FPS_30 = 30
+const val FPS_60 = 60
+
+// seconds duration in millis
+const val UNLIMITED_DURATION = -1L
```

Comment:
What about consistent the value's name?

Reply:
whoops, good catch on these, thank you :-)